src/Arrays/AppendedArrays.jl

```diff
@@ -59,11 +59,16 @@ struct AppendedArray{T,A,B} <: AbstractVector{T}
   a::A
   b::B
   function AppendedArray(a::AbstractArray,b::AbstractArray)
-    A = typeof(a)
```

Comment:
This version of the array constructor is actually the one that is in the main branch (as per my request). Thus, I would avoid adding these changes. These are a bit confusing. (Why there are git mods if we left the constructor untouched?)